tasks/beats-debian.yml

```diff
@@ -1,59 +1,78 @@
 ---
+- name: Debian - Ensure trusty dependencies are installed
+  block:
+    - name: Debian - Ensure trusty python dependencies are installed
+      apt:
+        name: "{{ item }}"
+        state: present
+      loop:
```

Comment:
This part if failing in centos-7 tests ([elastic+ansible-beats+pull-request/25/OS=centos-7,TEST_TYPE=standard](https://devops-ci.elastic.co/job/elastic+ansible-beats+pull-request/25/OS=centos-7,TEST_TYPE=standard,label=linux/console)) because centos-7 use ansible 2.4.2.0-2.el7 and `loop` is added in ansible 2.5 ([playbooks_loops](https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html)).

If you want to use `loop`, you should edit `.kitchen.yml` to install ansible using epel packages like we do for centos-6 [here](https://github.com/elastic/ansible-beats/blob/master/.kitchen.yml#L63-L64).

You would also need to update ansible required version to 2.5.0 in `README.md` and [meta/main.yml](https://github.com/elastic/ansible-beats/blob/master/meta/main.yml#L11).

Reply:
using `with_items` instead, in order to support older, but still maintained, ansible versions